tools/system_tests.bat

```diff
@@ -4,7 +4,7 @@ rem We don't need no spinner - mainly because we aren't running tox interactivel
 set TOX_PARALLEL_NO_SPINNER=1
 rem Run up to 3 environments in parallel, but still send the output to stdout
 rem We only run the tests if the tox file exists
-c:\python38\python.exe -m tox -c tox-system_tests.ini --parallel 3 --parallel-live
+python.exe -m tox -c tox-system_tests.ini --parallel 1 --parallel-live
```

Comment:
The reason 3 didn't work is because sometimes the agents try to create `junit` and `kibana` folders simultaneously and fail. We got around this by creating the folders before running the tests. See https://github.com/ni/nimi-python/blob/e12ac5b7854f8bdfbfdbc4e21afe9a17fb524aa7/tools/system_test_jobs.groovy#L94